### ISSUE TITLE
Add images to canvas

### ISSUE THREAD

**Issue by arunnambissan** [terrylinla/react-native-sketch-canvas#162]
Is it possible to select images from the phone gallery and add it to canvas?

**Comment by paraswattschicmic** [comment]
@arunnambissan  have you successfully linked to this library?

**Comment by arunnambissan** [comment]
No. I have implemented fetching the image URI of an image from the gallery. But didn't find a way to add inside the canvas.

**Comment by krishcdbry** [comment]
Any update on this ?

**Comment by creambyemute** [comment]
@arunnambissan @paraswattschicmic @krishcdbry 
```localSourceImage={this.state.imageEditorBackgroundImage}```

**Comment by yestay90** [comment]
@creambyemute , is you localSouceImage gets a little bit blurred when you take picture of text?

**Comment by dearsikandarkhan** [comment]
Can we add multiple images on canvas?
